frame/staking/src/testing_utils.rs

```diff
@@ -277,30 +277,55 @@ pub fn get_weak_solution<T: Trait>(
 /// worker code.
 pub fn get_seq_phragmen_solution<T: Trait>(
 	do_reduce: bool,
-) -> (Vec<ValidatorIndex>, CompactAssignments, ElectionScore, ElectionSize) {
+) -> (
+	Vec<ValidatorIndex>,
+	CompactAssignments,
+	ElectionScore,
+	ElectionSize,
+) {
 	let iters = offchain_election::get_balancing_iters::<T>();
 
 	let sp_npos_elections::ElectionResult {
 		winners,
 		assignments,
 	} = <Module<T>>::do_phragmen::<OffchainAccuracy>(iters).unwrap();
 
-	offchain_election::prepare_submission::<T>(assignments, winners, do_reduce).unwrap()
+	offchain_election::prepare_submission::<T>(
+		assignments,
+		winners,
+		do_reduce,
+		T::MaximumBlockWeight::get(),
```

Comment:
Why limit it here at all? Wouldn't that lead to a complete stall if the solution does compute more than `MaximumBlockWeight`?

Reply:
Again, this is only in the testing context, so I think it is fine to just set the limit to this amount. Unless if I am missing something?